README.md

```diff
@@ -18,21 +18,43 @@ Currently, the archetype is tested for Python 3.7, 3.8 and 3.9. It could work ou
 
 ## Getting started
 
-Clone the project using git, open a CLI in the cloned folder `python-archetype/src/archetype` and call:
+Clone the project using git, open a CLI in the cloned folder `python-archetype/src/archetype`.
 
-```
-python archetype.py
-```
-
-Afterwards you are prompted to input:
+The following information is required, regardless of wheter the CLI mode or the UI mode is used:
 * a project name
 * the target python version (e.g. 3.7, 3.8, 3.9, ...)
 * the base folder for the created project (basefolder + project name = source of generated project)
 
-Voila, you have created a clean python project.
+After adding the information, you have created a clean python project.
 
 To adapt or test the project have a look at the detailed description [here](./documentation/Tox.md)
 
+### CLI Mode
+
+In the CLI mode use the `archetype.py` script to create your target project using the following console arguments:
+- `-p`: The name of your project.
+- `-v`: The target Python version.
+- `-t`: The target base folder.
+- `-D`: If there is an overlapping project with same name in the targetfolder, delete it.
+
+Like:
+```python
```

Comment:
I think shell or bash makes more sense here as it is not really python code.

And I don't know, but should be state somewhere, that this has to be executed in the /src/archetype subfolder?